jupyterhub/app.py

```diff
@@ -996,6 +996,15 @@ def _cookie_secret_check(self, change):
         """,
     ).tag(config=True)
 
+    api_page_default_limit = Integer(
```

Comment:
For backward-compatibility, let's set both of these to 0 (no limit). That lets us make pagination opt-in for at least one release cycle.

Reply:
Can do 👍 Should that be done in this PR now that it's merged or somewhere else?